### ISSUE TITLE
0.3.1 fill-rule broke draw.line

### ISSUE THREAD

**Issue by Illicitline45** [cetz-package/cetz#730]
Title: when drawing a line ( `cetz.draw.line((0, 0), (0, 2))` for instance) it returns the error: " `dictionary does not contain key "fill-rule"` `".`

(`cetz.draw.line` was of course called inside `cetz.canvas({})` context, with no parameters given to canvas aside for the code block `{}` and its contents).

**Comment by johannes-wolf** [comment]
What was the problem? How can Typst throw this error in a cetz context?

**Comment by Illicitline45** [comment]
> What was the problem? How can Typst throw this error in a cetz context?

Typst is a little freaky (or maybe it's just me that is stupid) when it comes to contexts and imports. It used to be that I had a template with the func `cetz.canvas` imported and I could use it in a document just with `#canvas` where I had imported the custom template, but after I updated to typst 0.12 and cetz 0.3.1 it didn't work anymore, but just doing `#cetz.canvas` works.
I suspect it has something to do with what they did to contexts in the update, though when I first read the error message it threw I immediatly thought of the 0.3.1 patch notes and thought it was a cetz problem, sorry.
